packages/shared/src/utils/routes/routes.ts

```diff
@@ -109,6 +109,7 @@ export const DISABLE_LANDSCAPE_BLOCKER_ROUTES = [
     routes.account,
     routes.endpoint,
     routes.cashier,
+    routes.cashier_p2p,
```

Comment:
Is this required? Only `routes.cashier` will do right?

Reply:
yeah.. now that cashier route is added its not needed.  will udpate.